pkgs/by-name/sc/screenly-cli/package.nix

```
@@ -0,0 +1,48 @@
+{ darwin
+, fetchFromGitHub
+, lib
+, perl
+, pkg-config
+, openssl
+, rustPlatform
+, stdenv
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "screenly-cli";
+  version = "0.2.3";
+
+  src = fetchFromGitHub {
+    owner = "screenly";
+    repo = "cli";
+    rev = "refs/tags/v${version}";
+    hash = "sha256-rQK1EYb1xYtcxq0Oj4eY9PCFMoaYinr42W8NkG36ps0=";
+  };
+
+  cargoPatches = [
+    ./add-cargo-lock.patch
+  ];
+
+  cargoHash = "sha256-TzJ56Wuk77qrxDLL17fYEj4i/YhAS6DRmjoqrzb+5AA=";
+
+  nativeBuildInputs = [
+    pkg-config
+    perl
+  ];
+
+  buildInputs = [
+    openssl
+  ] ++ lib.optionals stdenv.isDarwin [
+    darwin.apple_sdk.frameworks.CoreFoundation
+    darwin.apple_sdk.frameworks.Security
+  ];
+
+  meta = {
+    description = "Command Line Interface (CLI) for Screenly.";
```

```suggestion
    description = "Command Line Interface for Screenly";
```
This doesn't explain what Screenly is.

---

Fixed!